### DISCUSSION TITLE
Error con mi contraseña de teams

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/CursoIntroPython#136] [Q&A] [answered]
Ya mande correo para poder cambiarla pero aun no me contesta nadie con quien puedo ver este tema

**Comment by mxgmr** [comment, marked as answer]
Puedes mandar un mensaje en el Instagram de innovaccion, en los lives dijeron que ahí estarían resolviendo esos problemas. Su Instagram es innovaccionvirtual

**Reply by gersondlarosa** [reply]
muchas gracias ya mande mensaje
